front-end/src/components/HeaderLogo.css

```diff
@@ -2,7 +2,12 @@
   display: block;
   margin-left: auto;
   margin-right: auto;
+<<<<<<< HEAD
```

Comment:
This looks like a merge artifact; it needs to be sorted out before we can merge. I get an error whenever I view a page.